spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,95 @@
+require_relative '../lib/board'
+require_relative '../lib/player'
+require_relative '../lib/game'
+
+describe Game do
+  let(:game) { Game.new }
+  let(:board) { Board.new }
+  let(:playerx) { Player.new('Player 1', 'X') }
+
+  describe '#welcome_player' do
+    it 'welcomes player' do
+      expect(game.welcome_player('A', 'X')).to eq("\nA will play with X\n\n")
+    end
+  end
+
+  describe '#display_board' do
+    it 'displays board' do
+      expect(game.display_board(board.board)).to eq("+---+---+---+\n|\
+   |   |   |\n+---+---+---+\n|   |   |   |\n+---+---+---+\n|   |\
+   |   |\n+---+---+---+\n\n")
+    end
+  end
+
+  # DRAW_board board = ['X','O','O','O','X','X','X','X','O']
+  # WIN_board_X board = ['X','O',' ',' ','X','O',' ',' ','X']
+  # WIN_board_X board = ['O','O','X',' ','X','O','X',' ',' ']
+  # WIN_board_O board = [' ',' ','X','O','O','O',' ',' ','X']
+  # NoWIN_NoDRAW_board_X board = [' ',' ',' ','X',' ',' ',' ',' ',' ']
```

Comment:
- Remove lines of commented code in your files.

Reply:
Done